api/test/common/diag/logLevel.test.ts

```diff
@@ -77,7 +77,7 @@ describe('LogLevelFilter DiagLogger', () => {
 
   const levelMap: Array<{
     message: string;
-    level: DiagLogLevel;
+    level: number | DiagLogLevel;
```

Comment:
This change is because this test is purposely using invalid enum values to check that the function handles them correctly. Typescript v4 didn't complain about invalid values, but Typescript v5 did so I added `number` to explicitly tell it we're using invalid numbers